pkg/cli/options.go

```diff
@@ -181,13 +181,29 @@ func isHostSupported(host string) bool {
 	return false
 }
 
-// getPluginsRoot detects the host system and gets the plugins root based on the host.
+// getPluginsRoot gets the plugin root path.
 func getPluginsRoot(host string) (pluginsRoot string, err error) {
 	if !isHostSupported(host) {
 		// freebsd, openbsd, windows...
 		return "", fmt.Errorf("host not supported: %v", host)
 	}
 
+	// if user provides specific path, return
+	if pluginsPath := os.Getenv("KB_PLUGINS_PATH"); pluginsPath != "" {
+		// verify if the path actually exists
+		if _, err := os.Stat(pluginsPath); err == nil {
+			// the path exists
+			return pluginsPath, nil
+		} else if os.IsNotExist(err) {
+			// the path does not exist
+			return "", fmt.Errorf("the specified path %s does not exist", pluginsPath)
+		} else {
+			// some other error
+			return "", fmt.Errorf("error checking the path: %v", err)
+		}
```

Comment:
Nit: Overall I think this looks fine, but I personally prefer a format more like:
```suggestion
		if _, err := os.Stat(pluginsPath); err != nil {
		    if os.IsNotExist(err) {
			// the path does not exist
			return "", fmt.Errorf("the specified path %s does not exist", pluginsPath)
		    }
		    // some other error
		    return "", fmt.Errorf("error checking the path: %v", err)
		} 
		// the path exists
		return pluginsPath, nil
```
No need to change this if you don't want to, just figured I would share my 2 cents as I think this condenses it into fewer lines of code that is a bit easier to read